tests/units/adapterSubjects.test.ts

```diff
@@ -36,10 +37,10 @@ describe("Adapter subjects", () => {
         )
       );
     });
-    it.only("marketsData", async () => {
```

Comment:
the only was unwanted (I think)

Reply:
*We should add a lint check to prevent that imo